mlos_bench/mlos_bench/config/schemas/config_schemas.py

```diff
@@ -100,6 +105,8 @@ def validate(self, config: dict) -> None:
         """
         # Allow skipping schema validation for tight dev cycle changes.
         # Note: this may cause pytest to fail if it's expecting exceptions to be raised for invalid configs.
```

Comment:
Move this comment up towards the env var along with a "See: validate()" or some such?